### PR TITLE
post-git-migration svn r4096 (escape_special_char.pro and file_move.pro)

### PR DESCRIPTION
**Pull request by slayoo**
changes by @alaingdl

### CODE REVIEW

**Comment by slayoo** [comment]
@alaingdl could you please comment if this has anything to do with this issue reported by @olebole:
https://sourceforge.net/p/gnudatalanguage/patches/97/
Thanks!

**Comment by alaingdl** [comment]
I would say no, but I need to check again

the problem in "test_file_move.pro" is we use a file with a '$' which may be consider
as an ENV (printenv) in Bash shell ...

**Comment by slayoo** [comment]
BTW, are these changes covered in the testsuite?

File: src/pro/escape_special_char.pro [hunk 1]
```diff
@@ -1,84 +1,108 @@
...
+if N_ELEMENTS(str_in) GT 1 then MESSAGE, 'input string should not be an array'
+;
 if KEYWORD_SET(verbose) then begin
-    MESSAGE, /continue, '/verbose keyword not available, please contribute'
+   MESSAGE, /continue, '/verbose keyword not available, please contribute'
 endif
 ;
 ; list of special characteres can be modified if it is not complete
-;if( not KEYWORD_SET(list_of_special_char)) then list_of_special_char ='[*.^$,&\]'
+;if( not KEYWORD_SET(list_of_special_char)) then list_of_special_char ='[*^$,&\]'
```
Comment (slayoo) [inline src/pro/escape_special_char.pro:60]: let's write a test for this change!

Comment (alaingdl) [reply src/pro/escape_special_char.pro:60]: those lines are commented ...
the real case in l47 vs l64 : removing "*" & ".", adding space " "

Comment (alaingdl) [reply src/pro/escape_special_char.pro:60]: as I said in the comments, the goal would be to *remove* this old code, and having in the C-code
using primitives (without "spawn")

**Comment by slayoo** [comment]
Merging as it anyhow is in the SF repo

**Pull request merged**